tests/systemd_testsuite/test_01_basic.pm

```diff
@@ -0,0 +1,25 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Run test executed by TEST-01-BASIC from upstream after openSUSE/SUSE patches.
+# Maintainer: Sergio Lindo Mansilla <slindomansilla@suse.com>, Thomas Blume <tblume@suse.com>
+
+use base "consoletest";
+use warnings;
+use strict;
+use testapi;
+use utils 'zypper_call';
+use power_action_utils 'power_action';
+
+sub run {
+    select_console 'root-console';
```

Comment:
Are you just providing skeleton or you haven't pushed implementation which matches the description?

Reply:
Just a skeleton. @tsaupe will provide the implementation with my support.
The test suite will be schedule on the development job group